### PR TITLE
Postman related file relocation and adding the postman mock server instruction

### PR DESCRIPTION
**Pull request by Eileen1003**
Created a folder named "Postman" and relocated the existing postman related files into it. Authored a file that provides basic instructions on how to use Postman mock server.

### CODE REVIEW

**Comment by Damngoblin** [comment]
Seems pretty clear for a first-time postman mock server user. It will better if you add a conclusion and additional resources. Also, please pay attention to the formatting (space between text and images) and don't forget to include references to avoid plagiarism.

File: Topics/Tech_Stacks/Postman/Postman_mock_server.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Postman Mock Server
```
Comment (Damngoblin) [inline Topics/Tech_Stacks/Postman/Postman_mock_server.md:1]: Seems pretty clear for a first-time postman mock server user.

File: Topics/Tech_Stacks/Postman/Postman_mock_server.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+3. Enter a name for the example.
+4. Edit the request part of the example.
+- Add any parameters or headers as needed.
+- Enter the request body and select a content type.
+5. Edit the response part of the example.
+- Enter a Status Code (such as 200 or 404).
+- Enter the response body and select a content type.
+- Add any headers as needed.
+6. Select Save to save the example.
+![](./assets/examples-add-custom.jpg)
```
Comment (Damngoblin) [inline Topics/Tech_Stacks/Postman/Postman_mock_server.md:98]: It will better if you add a conclusion and additional resources.

Comment (Damngoblin) [inline Topics/Tech_Stacks/Postman/Postman_mock_server.md:98]: start a new line between text and images

Comment (Eileen1003) [reply Topics/Tech_Stacks/Postman/Postman_mock_server.md:98]: Oh I forgot to include these. Thank you so much for your thoughtful suggestions. I will add these as suggested.

Comment (Eileen1003) [reply Topics/Tech_Stacks/Postman/Postman_mock_server.md:98]: Sure. I will reformat the document.

**Comment by Damngoblin** [comment]
Looks great now

**Pull request merged**